Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM rust:1.65.0-buster
```

Comment:
Is this the best image to start from? I predfer ubuntu based images when possible.

Reply:
Solved using multistage build.